### DISCUSSION TITLE
Block client-to-client traffic

### DISCUSSION THREAD

**Discussion by smi-ed** [strongswan/strongswan#1394] [Q&A]
How to block traffic between Strongswan VPN users?
So that everyone has access to the Internet but without the ability to ping each other.

Where can this be configured? At the config level `/etc/ipsec.conf` or in `iptables`?

**Comment by Thermi** [comment]
Use an iptables rule in *filter FORWARD and/or configure the local TS to exclude the subnet you assign the VIPs from.
